src/sql/parts/dashboard/dashboard.module.ts

```diff
@@ -124,5 +130,15 @@ export class DashboardModule {
 		this._bootstrap.selector = uniqueSelector;
 		(<any>factory).factory.selector = uniqueSelector;
 		appRef.bootstrap(factory);
+
+		this._router.events.subscribe(e => {
+			if (e instanceof NavigationStart) {
+				TelemetryUtils.addTelemetry(this._bootstrapService.telemetryService, TelemetryKeys.DashboardNavigated, {
+					uri: this._bootstrap.getUnderlyingUri(),
```

Comment:
can the URI have EUII in it?  like database, server names?

Reply:
I believe the the url can to cause it will have the full path of the executable path of sqlops in it. @AbbiePetcht verify

Reply:
I think route ID tell how many time it has navigate between pages. I will verify this

---
In reply to: [173600385](https://github.com/Microsoft/sqlopsstudio/pull/889#discussion_r173600385) [](ancestors = 173600385)

Reply:
I got rid of connection URI

---
In reply to: [173602929](https://github.com/Microsoft/sqlopsstudio/pull/889#discussion_r173602929) [](ancestors = 173602929)